utils/normalize/myList.js

```diff
@@ -0,0 +1,9 @@
+import { normalize, schema } from 'normalizr';
+
+export default (data) => {
+  const listSchema = new schema.Entity('list');
```

Comment:
Have you normalized movies which is in the list?

Reply:
No, I have not. Because we can't request custom list movies with total items, pages, and movies. 
API v3 does not give such an opportunity.

Reply:
But you need to store movies in place where you store it for favourites, watchlist etc.
Your list item should contain only ids

Reply:
I understood and fixed it.